flang/lib/Lower/CallInterface.cpp

```diff
@@ -223,15 +246,21 @@ bool Fortran::lower::CalleeInterface::hasAlternateReturns() const {
 }
 
 std::string Fortran::lower::CalleeInterface::getMangledName() const {
-
```

Comment:
NIT: This deletion is done by clang-format ? Never mind :)

Reply:
No, I think I mechanically removed it. The blank just struck me in my editor, but I probably should have just left it, it was OK.